.github/workflows/check_scala3_nightly.yml

```diff
@@ -7,10 +7,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: olafurpg/setup-scala@v13
+      - uses: coursier/setup-action@v1
+        with:
+          apps: ""
```

Comment:
Is this needed?

Reply:
I don't think so. I use this here as well https://github.com/scalameta/nvim-metals/blob/d5864d9395152d4820bcb8a66f1235c5970e46a5/.github/workflows/ci.yml#L63-L65 and don't need to have that.

Reply:
the default value of apps is `sbtn` but there is no need to install it as for this job only jvm and cs is required

Reply:
Ahh cool! I didn't know that.